fe/fe-core/src/main/java/com/starrocks/persist/gson/GsonUtils.java

```diff
@@ -269,6 +273,11 @@ public class GsonUtils {
             .registerTypeHierarchyAdapter(Table.class, new GuavaTableAdapter())
             .registerTypeHierarchyAdapter(Multimap.class, new GuavaMultimapAdapter())
             .registerTypeAdapterFactory(new ProcessHookTypeAdapterFactory())
+            // specified childcalss DeSerializer must be ahead of fatherclass,
```

Comment:
Is it means that subtype defined in COLUMN_TYPE_ADAPTER_FACTORY should be declared ahead of COLUMN_TYPE_ADAPTER_FACTORY?

Reply:
yes